sdk/nodejs/runtime/mocks.ts

```diff
@@ -155,7 +155,7 @@ export class MockMonitor {
 
     public async readResource(req: any, callback: (err: any, innterResponse: any) => void) {
         try {
-            let custom = false;
+            let custom = true;
             if (typeof req.getCustom === "function") {
                 custom = req.getCustom();
             }
```

Comment:
I don't imagine this is needed.

Reply:
:+1: I'm inclined to agree. Will remove.